cluster-provision/k8s/1.27/k8s_provision.sh

```diff
@@ -161,8 +161,8 @@ envsubst < $kubeadm_raw_ipv6 > $kubeadm_manifest_ipv6
 until ip address show dev eth0 | grep global | grep inet6; do sleep 1; done
 
 # workaround for 1.25 trying to pull coredns: https://github.com/kubernetes/kubernetes/issues/112131#issuecomment-1232279438
-podman pull k8s.gcr.io/coredns/coredns:v1.9.3
-podman image tag k8s.gcr.io/coredns/coredns:v1.9.3 k8s.gcr.io/coredns:v1.9.3
+podman pull k8s.gcr.io/coredns/coredns:v1.10.1
+podman image tag k8s.gcr.io/coredns/coredns:v1.10.1 k8s.gcr.io/coredns:v1.10.1
```

Comment:
Can we derive the tag needed automatically, so we don't need to maintain this code maybe?

Don't we need to update extra-pre-pull-images ?
In the past we had to IIRC, maybe the mechanism was changed